doc/user/content/connect-sources/cdc-postgres-direct.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "PostgreSQL CDC"
+title: "Ingest data from PostgreSQL"
 description: "How to propagate Change Data Capture (CDC) data from a PostgreSQL database to Materialize"
```

Comment:
In PostgreSQL, this is more often referred to a "replication" (e.g. logical replication) than CDC.